src/components/Gallery/index.tsx

```diff
@@ -333,7 +333,7 @@ const Gallery: React.FC<GalleryProps> = ({
     onSwiping: (event) => {
       setSlide({
         isSliding: true,
-        delta: event.deltaX,
+        delta: -event.deltaX,
```

Comment:
invert the delta x per the breaking change and inversion in the library https://github.com/FormidableLabs/react-swipeable/blob/main/migration.md#major-changes